vcs/repository_test.go

```diff
@@ -1673,7 +1673,7 @@ func TestRepository_UpdateEverything(t *testing.T) {
 		}
 
 		// update the mirror.
-		err = r.(vcs.RemoteUpdater).UpdateEverything(vcs.RemoteOpts{})
+		_, err = r.(vcs.RemoteUpdater).UpdateEverything(vcs.RemoteOpts{})
```

Comment:
It would be better if this test were updated to also check the value of `UpdateResult`. This would be in addition to the unit tests for the update result parser.